src/core/highlight/behaviors/cell.highlight.js

```diff
@@ -9,10 +9,10 @@ export default class CellHighlight extends Behavior {
 	applyCore(items) {
 		const body = this.markup.body;
 		for (let item of items) {
-			if (body && body.rows && body.rows.length > item.rowIndex) {
-				const row = body.rows[item.rowIndex];
-				if (row && row.cells && row.cells.length > item.columnIndex) {
-					this.state(row.cells[item.columnIndex], true);
+			if (body && body.rows && body.rows.length > item.parentElement.rowIndex) {
```

Comment:
can you add a namew to item.parentElement? it's not recognizable from context?
Don't need to check for body && body rows i gues?